core/src/apps/webauthn/fido_seed.py

```
@@ -0,0 +1,18 @@
+        from trezor.crypto import se_thd89
+        from utime import sleep_ms
+
+        while True:
+            try:
+                ret = se_thd89.fido_seed()
+                if ret:
+                    break
+                else:
+                    sleep_ms(100)
+                    continue
+            except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)